guide/source/testing.md

```diff
@@ -238,19 +238,22 @@ import { Tracker } from 'meteor/tracker';
 const withDiv = function withDiv(callback) {
   const el = document.createElement('div');
   document.body.appendChild(el);
+  let view = null
   try {
-    callback(el);
+    view = callback(el);
   } finally {
+    if (view) Blaze.remove(view)
     document.body.removeChild(el);
   }
 };
 
 export const withRenderedTemplate = function withRenderedTemplate(template, data, callback) {
   withDiv((el) => {
     const ourTemplate = isString(template) ? Template[template] : template;
-    Blaze.renderWithData(ourTemplate, data, el);
+    vonst view = Blaze.renderWithData(ourTemplate, data, el);
```

Comment:
Typo here

```suggestion
    const view = Blaze.renderWithData(ourTemplate, data, el);
```

please make sure this code runs properly; many users just copy and paste from tutorials.

Reply:
oops, done.